ios/Classes/SmileIDPlugin.swift

```
@@ -282,4 +283,98 @@ public class SmileIDPlugin: NSObject, FlutterPlugin, SmileIDApi {
             })
             .store(in: &subscribers)
     }
+    
+    
+    func pollSmartSelfieJobStatus(
+        request: FlutterJobStatusRequest,
+        interval: Int64,
+        numAttempts: Int64,
+        completion: @escaping (Result<FlutterSmartSelfieJobStatusResponse, Error>) -> Void
+    ) {
+        
+        SmileID.api.pollSmartSelfieJobStatus(request: request.toRequest(),
+                                             interval: TimeInterval(interval),
+                                             numAttempts: Int(exactly: numAttempts)!)
+        .sink(receiveCompletion: { status in
+            switch status {
+            case .failure(let error):
+                completion(.failure(error))
+            default:
```

What are the other cases here? Are we going to end up silently swallowing the result?